README.md

```diff
@@ -82,6 +82,37 @@ class TestAppDelegate: SpeziAppDelegate {
 }
 ```
 
+[SpeziLLMLocalDownload](https://swiftpackageindex.com/stanfordspezi/spezillm/documentation/spezillmlocaldownload) can be used to download an LLM from [HuggingFace](https://huggingface.co/) and save it on the device for execution. The `LLMLocalDownloadView` provides an out-of-the-box onboarding view for downloading models locally.
+
+```swift
+struct LLMLocalOnboardingDownloadView: View {
+    @Environment(OnboardingNavigationPath.self) private var onboardingNavigationPath
+
+    var body: some View {
+        LLMLocalDownloadView(
+            model: .llama3_8B_4bit,
+            downloadDescription: "The Llama3 8B model will be downloaded",
+        ) {
+            onboardingNavigationPath.nextStep()
+        }
+    }
+}
+```
+
+This view can then be included in your onboarding process using [SpeziOnboarding](https://swiftpackageindex.com/stanfordspezi/spezionboarding/documentation).
+
+```swift
+struct LLMLocalDownloadApp: View {
+    @State private var path = NavigationPath()
+
+    var body: some View {
+        NavigationStack(path: $path) {
+            LLMLocalOnboardingDownloadView()
+        }
+    }
+}
+```
+
```

Comment:
Wondering if this example should be removed or actually demonstrate the usage of the view in a Spezi Onboarding context, it currently doesn't use any onboarding functionality. I would lean toward removing it.

Reply:
You make a valid point. This example can go and we will point people to the SpeziOnboarding documentation here instead.